code/__HELPERS/grammar.dm

```diff
@@ -0,0 +1,37 @@
+/// Returns a list in plain english as a string
+/proc/english_list(list/input, nothing_text = "nothing", and_text = " and ", comma_text = ", ", final_comma_text = "" )
```

Comment:
moving it is pointless, it fits helpers for lists

Reply:
I believe this makes sense to be here, and I am going to add more grammar correction procs there.

Reply:
now moved to `text.dm`

Reply:
maybe out of the scope for the main topic of this PR